utility/src/main/java/org/oppia/android/util/parser/image/BUILD.bazel

```
@@ -1,5 +1,5 @@
 """
-bazel file fofr image in parser
+General purpose utilities for image loading and processing
```

```suggestion
General-purpose utilities for image loading and processing.
```

Ditto for other doc strings.

---

fixed the doc strings for all bazel files